synapse/cores/common.py

```diff
@@ -1938,3 +1912,68 @@ def _tufosByInetCidr(self, prop, valu, limit=None):
 
         return self.getTufosBy('range', prop, (ipv4addr, ipv4addr+mask), limit=limit)
 
+    #############################################################
```

Comment:
general discussion to have: type hierarchy of a cortex is a little weird. cortex -> datamodel -> typelib. 

conceptually, cortex has-a datamodel, *not* cortex is-a datamodel. this suggests the datamodel might be a better fit as a member of the cortex, rather than a parent class.

practically, this means i had to override these methods to shim in the persistence.